packages/react-bindings/src/styles/calculateAnimationTimeout.ts

```diff
@@ -0,0 +1,10 @@
+import convertCssTimeToNumber from './convertCssTimeToNumber'
+import * as _ from 'lodash'
+
+const calculateAnimationTimeout = (duration: string, delay: string | undefined): number => {
```

Comment:
isn't `delay: string | undefined` equivalent with `delay?: string` ?

Reply:
No, there are not equivalent

```
(delay?: string) === (delay: string | undefined = undefined)
```

https://www.typescriptlang.org/play/#code/GYVwdgxgLglg9mABAQwBTDnA-ALkQZygCcYwBzASkQG9EBfAKAbQudQHIM53WHRJYCRACN0mPIRLlEAH0TgAJgFNgpJQqq1Go1qM6YeDIA

Reply:
I guess my question was more like "why would I ever pass explicitly `undefined` as the last parameter" of a function?

Reply:
Because we are passing props here and the prop can be undefined... Anyway, I agree that for the last param it can be optional, for the other ones we need `string | undefined`